src/frontend/Komponenter/Behandling/Brevmottakere/SkalBrukerHaBrev.tsx

```diff
@@ -1,11 +1,10 @@
 import React, { Dispatch, FC, SetStateAction } from 'react';
-import { Ingress } from 'nav-frontend-typografi';
-import { Radio, RadioGruppe } from 'nav-frontend-skjema';
 import { IPersonopplysninger } from '../../../App/typer/personopplysninger';
 import { EBrevmottakerRolle, IBrevmottaker } from './typer';
 import styled from 'styled-components';
+import { Ingress, Radio, RadioGroup } from '@navikt/ds-react';
 
-const StyledRadioGruppe = styled(RadioGruppe)`
+const StyledRadioGruppe = styled(RadioGroup)`
```

Comment:
```suggestion
const RadioGruppe = styled(RadioGroup)`
```

En grei retningslinje er å gi stylede komponenter mest mulig semantisk riktige navn :)

Reply:
Evt. `RadioKnapper`